src/mococrw/mac.h

```diff
@@ -145,4 +145,80 @@ class HMAC : public MessageAuthenticationCode {
     std::unique_ptr<Impl> _impl;
 };
 
+class CMAC : public MessageAuthenticationCode {
+public:
+    /**
+     * @brief Constructor
+     * @param hashFunction The hash function which shall be used
+     * @param key The key used for CMAC
+     */
+    CMAC(mococrw::openssl::CmacCipherTypes cipherType, const std::vector<uint8_t> &key);
+
+    /**
+     * @brief destructor
+     */
+    ~CMAC();
+
+    /**
+     * Adds the chunk of data to the hash. H(i-key || message1 [ || message2])
```

Comment:
```suggestion
     * Adds the chunk of data to the MAC.
```

I believe `H(i-key || message1 [ ||  message2 ])` is the formula for HMAC, is it not?

Also, hashing isn't involved in AES-CMAC, so let's not use the word "hash".

Reply:
Yes, sorry. I did miss the comments of this header :(